### ISSUE TITLE
error: Invalid comparator

### ISSUE THREAD

**Issue by mesqueeb** [raineorshine/npm-check-updates#660]
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce

I have a project with some forks which I use directly from github.
So my package.json looks like this:

```
  "signature_pad": "github:mesqueeb/signature_pad",
```

## Current Behavior

Execute 
- `ncu --semverLevel major`

throws error:

```
lucaban@Golden-Birdie ~/.g/g/m/sokketsu (dev) [1]> ncu --semverLevel major
Checking /Users/lucaban/.ghq/github.com/mesqueeb/sokketsu/package.json
[=====---------------] 10/41 24%/usr/local/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:415
    throw err;
    ^

TypeError: Invalid comparator: github:mesqueeb/signature_pad
```

## Expected Behavior

It should ignore such packages.

**Comment by raineorshine** [comment]
Thanks! Will publish a patch later today.

**Comment by raineorshine** [comment]
Published in `v6.0.1`.
